modules/GKB/Release/Steps/OrthoInference.pm

```diff
@@ -5,46 +5,61 @@ use GKB::Config_Species;
-has '+mail' => ( default => sub { 
-                        my $self = shift;
-                        return {
-                            'to' => '',
-                            'subject' => $self->name,
-                            'body' => "",
-                            'attachment' => ""
-                        };
+has '+mail' => ( default => sub {
+    my $self = shift;
+    return {
+        'to' => '',
+        'subject' => $self->name,
+        'body' => '',
+        'attachment' => ''
+    };
 });
 has '+user_input' => (default => sub { {
-                    'skip_list_verified' => {'query' => "Has the normal event skip list been verified for version $version (y/n):"},
-                    'release_date' => {'query' => "Enter release date for version $version as yyyy-mm-dd (e.g. 2017-12-13):" }
+    'skip_list_verified' => {'query' => "Has the normal event skip list been verified for version $version (y/n):"},
+    'release_date' => {'query' => "Enter release date for version $version as yyyy-mm-dd (e.g. 2017-12-13):" },
+    'person_id' => {'query' => 'Enter person instance db id to be used for the orthoinference instance edit:'},
 } });
 
 override 'run_commands' => sub {
     my ($self, $gkbdir) = @_;
 
+    my $host = $self->host;
+
     my $skip_list_verified = $self->user_input->{'skip_list_verified'}->{'response'} =~ /^y/i;
     if (!$skip_list_verified) {
-        die "Skip list must be verified before running the orthoinference process";
+        confess "Skip list must be verified before running the orthoinference process\n";
     }
 
     my $release_date = $self->user_input->{'release_date'}->{'response'};
     if (!$release_date || $release_date !~ /^\d{4}-\d{2}-\d{2}$/) {
-        die "Release date for version $version needed as yyyy-mm-dd";
+        confess "Release date for version $version needed as yyyy-mm-dd\n";
     }
 
-    $self->cmd("Creating orthopredictions and backing up database",
+    my $person_id = $self->user_input->{'person_id'}->{'response'};
+    if (!$person_id || $person_id !~ /^\d+$/) {
+        confess "Person instance db id for instance edit is needed\n";
+    }
+
+    $self->cmd('Creating orthopredictions and backing up database',
         [
             ["mkdir -p $version"],
-            ["perl wrapper_ortho_inference.pl -r $version -user $user -pass $pass -port 3306 -release_date $release_date > $version/wrapper_ortho_inference.out"],
-            ["rm -f ../website_files_update/report_ortho_inference.txt"],
+            ["perl create_orthoinference_db.pl -source_db $slicedb -target_db $db" . 
+             " > create_orthoinference_db.out 2> create_orthoinference_db.err"],
+            ["perl build_orthoinference_jar.pl -release $version -release_date $release_date -person_id $person_id" . 
+             " > build_orthoinference_jar.out 2> build_orthoinference_jar.err"],
+            ['./runOrthoinference.sh > runOrthoinference.out 2> runOrthoinference.err'],
```

Comment:
This is where the new 'remove_unused_PE' script will go for next release. Perhaps a placeholder comment just for safety's sake?